packages/admin/cms-admin/src/dam/FileForm/FileSettingsFields.tsx

```diff
@@ -59,6 +59,18 @@ export const FileSettingsFields = ({ isImage, folderId }: SettingsFormProps): Re
         [apollo, folderId, scope],
     );
 
+    const requiredValidator = React.useCallback(
+        (value: unknown, allValues: object) => {
+            const type = (allValues as EditFileFormValues).license?.type;
```

Comment:
```suggestion
        (value: unknown, allValues: EditFileFormValues) => {
            const type = allValues.license?.type;
```

Is this not possible?

Reply:
Sadly that doesn't work but causes this TS error:

<img width="830" alt="Bildschirmfoto 2023-12-21 um 14 19 47" src="https://github.com/vivid-planet/comet/assets/13380047/d8c38a25-d73b-4bcd-8f71-d2e57f95a5ba">